aiohttp/client_reqrep.py

```diff
@@ -206,7 +220,7 @@ class ClientRequest:
     }
 
     # Type of body depends on PAYLOAD_REGISTRY, which is dynamic.
-    body: Any = b""
+    _body: Union[bytes, payload.Payload] = b""
```

Comment:
We should initialize to None as it makes it so much more sane

Reply:
The refactor initialises to an empty BytesPayload..